src/brogue/RogueMain.c

```diff
@@ -905,13 +904,16 @@ static void removeDeadMonstersFromList(creatureList *list) {
         if (decedent->bookkeepingFlags & MB_HAS_DIED) {
             removeCreature(list, decedent);
             if (decedent->leader == &player
-                && !(decedent->info.flags & MONST_INANIMATE)
-                && (decedent->bookkeepingFlags & MB_HAS_SOUL)
+                && !(decedent->info.monsterID == MK_PHOENIX)
+                && !(decedent->info.monsterID == MK_LICH)
```

Comment:
does vampire need to be included here too? it's mentioned in another comment

Reply:
The vampire is like the phylactery and phoenix egg in that it does the summoning. That has me thinking though, and I'll look into suppressing specifically summoned vampire bats, but not normal vampire bats.

It's worth noting that vampire resurrection was working before, the difference with this change is that upon revival, negation is removed and its ability to summon is restored.

Reply:
Ok I pushed an update that handles vampire bats and removes the hard-coded lich/phantom handling. As a side benefit, it also handles liches a little better. Not only will the phylactery be revived, but any furies or phantoms summoned by the lich will not be eligible for resurrection. So if your lich is still alive and you activate a resurrection altar, you won't get a random fury or phantom.